LICENSE

```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2017 Oracle Corporation
+   Copyright {yyyy} {name of copyright owner}
```

Comment:
Is this correct? Does it get expanded out as part of a commit hook or the build?

Reply:
Confirmed this is intentional and apparently what we want.